pkg/apis/core/validation/validation.go

```diff
@@ -5530,7 +5530,6 @@ func ValidateResourceQuantityValue(resource string, value resource.Quantity, fld
 // newResourceQuota is updated with fields that cannot be changed.
 func ValidateResourceQuotaUpdate(newResourceQuota, oldResourceQuota *core.ResourceQuota) field.ErrorList {
 	allErrs := ValidateObjectMetaUpdate(&newResourceQuota.ObjectMeta, &oldResourceQuota.ObjectMeta, field.NewPath("metadata"))
-	allErrs = append(allErrs, ValidateResourceQuotaSpec(&newResourceQuota.Spec, field.NewPath("spec"))...)
```

Comment:
ObjectMeta and Spec are different, i don't think ValidateResourceQuotaSpec should be skipped here.

Reply:
The detailed reason is in https://github.com/kubernetes/kubernetes/pull/98201#issuecomment-765116930.

The duplicate lines are Line 5533  and Line 5477. Both are spec validation.

Line 5533 here:
	allErrs = append(allErrs, ValidateResourceQuotaSpec(&newResourceQuota.Spec, field.NewPath("spec"))...)
Line 5477:
	allErrs = append(allErrs, ValidateResourceQuotaSpec(&resourceQuota.Spec, field.NewPath("spec"))...)